test/e2e/kubernetes/pvc/pvc.go

```diff
@@ -51,22 +52,34 @@ spec:
       storage: 5Gi
  */
 
+/*
+---
```

Comment:
Is this here as an example? If so maybe we can add a header comment describing this case and the case above ^.